operations/CSM_product_management/Post_Install_Customizations.md

```diff
@@ -291,76 +238,41 @@ This example is based on what was needed for a system with 4000 compute nodes.
-   yq write -i customizations.yaml 'spec.kubernetes.services.spire.cray-postgresql.sqlCluster.resources.limits.cpu' --style=double '8'
-   yq write -i customizations.yaml 'spec.kubernetes.services.spire.cray-postgresql.sqlCluster.resources.limits.memory' '8Gi'
-   ```
+| Chart name           | Base manifest name   | Resource path name | Kubernetes namespace |
+| -------------------- | -------------------- | ------------------ | -------------------- |
+| `cray-sls-postgres`  | `core-services`      | `cray-hms-sls`     | `services`           |
+| `cray-smd-postgres`  | `core-services`      | `cray-hms-smd`     | `services`           |
+| `gitea-vcs-postgres` | `sysmgmt`            | `gitea`            | `services`           |
+| `spire-postgres`     | `sysmgmt`            | `spire`            | `spire`              |
 
-1. (`ncn-mw#`) Check that the customization file has been updated.
+The path to the chart file is `${PATH_TO_RELEASE}/helm`. For the necessary customizations
+to make, and the steps to validate that the redeploy was successful, see the next two
+subsections.
 
-   ```bash
-   yq read customizations.yaml 'spec.kubernetes.services.spire.cray-postgresql.sqlCluster.resources'
-   ```
+#### Postgres customization changes
 
-   Example output:
+1. (`ncn-mw#`) Set the `rpname` variable to the appropriate resource path name from the table above.
 
-   ```yaml
-   requests:
-     cpu: "4"
-     memory: 4Gi
-   limits:
-     cpu: "8"
-     memory: 8Gi
-   ```
-
-1. (`ncn-mw#`) Edit the `sysmgmt.yaml` to only include the `spire` chart and all its current data, and change the `metadata.name` field to `spirepg`.
-
-   > **NOTE:** If you leave the `metadata.name` field unchanged, it will override the existing cached product manifest so please make sure
-   > to change it to `spirepg` before proceeding.
-
-   The resources specified above will be updated in the next step. The version may differ, because this is an example.
-
-   ```yaml
-   apiVersion: manifests/v1beta1
-   metadata:
-     name: spirepg
-   spec:
-     charts:
-     - name: spire
-       namespace: spire
-       values:
-       - "..."
-       version: 0.9.1
-   ```
+    ```bash
+    rpname=<put resource path name from table here>
+    ```
 
-1. (`ncn-mw#`) Generate the manifest that will be used to redeploy the chart with the modified resources.
+1. (`ncn-mw#`) Edit the customizations by adding or updating `spec.kubernetes.services.${rpname}.cray-postgresql.sqlCluster.resources`.
 
    ```bash
-   manifestgen -c customizations.yaml -i sysmgmt.yaml -o manifest.yaml
+   yq write -i customizations.yaml "spec.kubernetes.services.${rpname}.cray-postgresql.sqlCluster.resources.requests.cpu" --style=double '4'
+   yq write -i customizations.yaml "spec.kubernetes.services.${rpname}.cray-postgresql.sqlCluster.resources.requests.memory" '4Gi'
+   yq write -i customizations.yaml "spec.kubernetes.services.${rpname}.cray-postgresql.sqlCluster.resources.limits.cpu" --style=double '8'
```

Comment:
I'll also note that it's common practice now to not add a cpu limit on pods. Since we do this everywhere, I'm unsure if we should change anything here without having a larger discussion on right-sizing pods.

Reply:
Definitely a separate discussion. This PR is only for fixing the procedure we follow when telling customers to redeploy charts. That is already going to be extensive enough without trying to pull in other stuff. :)